servers/rendering/rasterizer_rd/shaders/scene_high_end.glsl

```diff
@@ -1889,6 +1892,10 @@ FRAGMENT_SHADER_CODE
 		specular_light *= scene_data.ambient_light_color_energy.a;
 	}
 
+	if (custom_radiance.a > 0.0) {
```

Comment:
I would use a define for this, making the shader fatter (more vgprs used) makes no sense imo.
like
#ifdef CUSTOM_RADIANCE_USED
code
#endif

check how other similar built-ins generate these ifdefs